.github/workflows/test.yaml

```
@@ -5,7 +5,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [1.14.x, 1.15.x]
+        go-version: [1.21.x]
```

nit: To apply this change, it needs to swap `Install Go` and `Checkout code` so `setup-go` can read go.mod.

```suggestion
        go-version-file: go.mod
```

---

updated